src/sentry/api/endpoints/integrations/sentry_apps/installation/external_requests.py

```diff
@@ -25,7 +25,7 @@ def get(self, request: Request, installation) -> Response:
         }
 
         if project:
-            kwargs.update({"project": project})
+            kwargs.update({"project_slug": project.slug})
```

Comment:
`SelectRequest` now takes`project_slug`, which is the only part of theproject it requires.  Reduces cross silo interactions (no need for entire ORM object).